.env

```diff
@@ -1,13 +1,14 @@
 HOST_PORT=8000
 
-DATABASE_HOST="db"
+DATABASE_HOST=db
 DATABASE_PORT=5432
-DATABASE_NAME="mydb"
+DATABASE_NAME=mydb
 DATABASE_USER=postgres
 DATABASE_PASSWORD=postgres
+POSTGRES_PASSWORD=${DATABASE_PASSWORD}
 
-CELERY_BROKER_URL='redis://localhost:6379/0'
+CELERY_BROKER_URL=redis://localhost:6379/0
 FLOWER_PORT=5555
 
-SECRET_KEY="django-insecure-*yfb&*wtp(t00s-31-_p&dgq*)$0h=%16gwi$d-45wpab4lt!6"
```

Comment:
You should take care about the reserved characters on passswords inside the `.env` 
There are better ways to manager secrect keys. Keep it in a file is not safe.